geometry_msgs/PoseStamped task
---
bool result